### PR TITLE
New File added

### PR DESCRIPTION
**Pull request by sushant-arora**
Please re-review
[file1.txt](https://github.com/sushant-arora/PULL/files/7020552/file1.txt)

### CODE REVIEW

File: Example [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (aliguzelaccenture) [inline Example]: needs 1 more line

**Review by aliguzelaccenture — CHANGES_REQUESTED** [review]
please add 1 more blank line lol

File: Example (hunk 1 above)
Comment (sushant-arora) [reply Example]: where do you want the extra line sire ?

**Review by aliguzelaccenture — COMMENTED** [review]
love the changes, you're promoted to lead developer

**Pull request merged**